rcl_interfaces/msg/ParameterValue.msg

```diff
@@ -6,8 +6,8 @@
 uint8 type
```

Comment:
:+1: sorry forgot about that, yeah it would be great if github allowed to enable notification for "reactions"